### PR TITLE
Metrics

### PR DESCRIPTION
**Pull request by kaiwirt**
Draft section on metrics, including the tables from github

### CODE REVIEW

File: guide/sections/wis-metrics.adoc [hunk 1]
```diff
@@ -4,4 +4,69 @@ The availability of data and performance of system components within WIS2 are ac
 
 WIS2 requires that metrics are provided using OpenMetrics – the de-facto standard footnote:[OpenMetrics is proposed as a draft standard within IETF.] for transmitting cloud-native metrics at scale. Widely adopted, many commercial and open-source software components already come preconfigured to provide performance metrics using the OpenMetrics standard. Tools such as Prometheus and Grafana provide aggregation and visualisation of metrics provided in this form, making it simple to generate performance insights.
 
-The content and structure of metrics specific to WIS2 (e.g., data availability) are described below.
+The roles of centers with respect to the monitoring as well as the content and structure of metrics specific to WIS2 (e.g., data availability) are described below.
```
Comment (tomkralidis) [inline guide/sections/wis-metrics.adoc:7]: centers -> centres ("centre" is being used across the document).

File: guide/sections/wis-metrics.adoc [hunk 2]
```diff
@@ -4,4 +4,69 @@ The availability of data and performance of system components within WIS2 are ac
...
+
+=== Roles
+
+==== Global Monitoring (GM)
+
+The Global Monitoring (Centers) are the entry points for users and provide the monitoring results. The main task of the Global Monitoring is to regularly query the provided metrics from the relevant WIS2 entities, aggregate and process the data and then provide the results to the end user in a suitable presentation.
```
Comment (tomkralidis) [inline guide/sections/wis-metrics.adoc:13]: Centers -> Centres

File: guide/sections/wis-metrics.adoc [hunk 3]
```diff
@@ -4,4 +4,69 @@ The availability of data and performance of system components within WIS2 are ac
...
+
+The WIS2 Global Services, namely the Global Broker, Global Cache, Global Discovery Catalogue and so on expose monitoring metrics on their respective service to the Global Monitoring. See <<wis2-metric-hierarchy>> for further details.
+
+==== WIS2 Nodes
+
+There is no requirement on WIS2 Nodes to provide monitoring metrics. However their WIS2 interfaces may be queried remotely by Global Services, which in turn can provide metrics on the availability of WIS2 Nodes.
+
+==== Sensor Centers and WMO programs
+
+WMO Programs can define monitoring requirements in order to be able to verify if the program's requirements are met. To do so, they need to define the relevant metrics. Metrics not directly related to the core services of WIS2 can be exported by so called Sensor Centers. Sensor Centers can read data or query services defined by the relevant WMO Program and provide the results in OpenMetrics format.
```
Comment (tomkralidis) [inline guide/sections/wis-metrics.adoc:25]: Centers -> Centres

Comment (tomkralidis) [inline guide/sections/wis-metrics.adoc:25]: Should there be a link to openmetrics.io (or in an Annex)?

File: guide/sections/wis-metrics.adoc [hunk 4]
```diff
@@ -4,4 +4,69 @@ The availability of data and performance of system components within WIS2 are ac
...
+
+==== Global Services
+
+The WIS2 Global Services, namely the Global Broker, Global Cache, Global Discovery Catalogue and so on expose monitoring metrics on their respective service to the Global Monitoring. See <<wis2-metric-hierarchy>> for further details.
+
+==== WIS2 Nodes
+
+There is no requirement on WIS2 Nodes to provide monitoring metrics. However their WIS2 interfaces may be queried remotely by Global Services, which in turn can provide metrics on the availability of WIS2 Nodes.
+
+==== Sensor Centers and WMO programs
```
Comment (tomkralidis) [inline guide/sections/wis-metrics.adoc:23]: Centers -> Centres

**Review by tomkralidis — APPROVED** [review]

**Review by tomkralidis — CHANGES_REQUESTED** [review]
Minor changes requested.

**Comment by kaiwirt** [comment]
Renamed to centre and added a reference to openmetrics.io

**Review by tomkralidis — APPROVED** [review]

**Pull request merged**